src/io/process.jl

```diff
@@ -236,48 +236,41 @@ function getMetaData(f)
     meta = f.params
     metaKeys = keys(meta)
     channel_properties = []
-    defaultValue = "None"
+    defaultValue = ""
 
     # determine the number of channels
     pars = parse(Int, strip(join(meta["\$PAR"])))
 
-    # determine the range of channel numbers
-    channel_numbers = 1:pars
-
-    # determine the channel properties
+    # determine the available channel properties
     for (key,) in meta
-        if key[1:3] == "\$P1"
-            if !occursin(key[4], "0123456789")
-                push!(channel_properties, key[4:end])
+        if key[1:2] == "\$P"
+            i=3
+            while i<=length(key) && isdigit(key[i])
+                i+=1
+            end
+            if i>length(key)
```

Comment:
I am not sure this is necessary.

Reply:
It is for scanning all possible parameter descriptions, which should generally be done right here. The original version in fact didn't scan single-number parameters, which was the reason why it failed on the flow FCS files (that have "problematic" columns 1-6).

Reply:
sure, but I am talking about the `continue`. I am sure there is another way to write the `if` conditions 😄

Reply:
...but `continue` is tail-generic! :D

Simplified in the commit below.